src/pages/CategoriesPage.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function CategoriesPage() {
+  return (
+    <div>
+      <div className="categories-wrapper row">
+        <button type="button">Check status</button>
+      </div>
+    </div>
+  );
+}
+
+export default CategoriesPage;
```

Comment:
- [ ] Kindly make sure that when the check status button is clicked, an action is dispatched in the redux store. like this example below 👇 

```.jsx

import React from "react"
import { useDispatch } from "react-redux"
import { checkStatus } from "path/to/categoriesSlice"

function Categories() {
  const dispatch = useDispatch()

  return (
    <div>
      <Button onClick={() => dispatch(checkStatus("Under construction"))}>
        Check status
      </Button>
      {/* Rest of Categories JSX */}
    </div>
  )
}

export default Categories
```